docs/element-type-is-invalid.md

```diff
@@ -15,13 +15,13 @@ This likely means that:
 This is a common mistake. Please see Melange's [Import an ES6 Default Value](https://melange.re/v2.0.0/communicate-with-javascript/#default-es6-values). Aka, instead of:
```

Comment:
I saw you updated the link to point to v4 above. we should likely update the melange doc links across the entire docs to be consistent, in that case.